pkg/daemon/update.go

```diff
@@ -1605,11 +1610,11 @@ func (dn *Daemon) writeFiles(files []ign3types.File) error {
 }
 
 func origParentDir() string {
-	return filepath.Join("/etc", "machine-config-daemon", "orig")
+	return origParentDirPath
```

Comment:
I think you did this for testing (?), but is there a reason you can't just mock the return value of this func? instead of having the var and func both existing at the same time?

Reply:
Yes, I am testing to make sure createOrigFile() is working properly, and I need a way to get it to write its files to a temporary directory during the test, but the file locations are essentially hard-coded because of those function calls.

Maybe I shouldn't worry about the test (or maybe it should be an e2e test because it does file i/o?). I don't know where you want to draw the line. 

Anyway, my options as I saw them were: 
	

1. make origParentDir/noOrigParentDir function pointers and then mock/override their functions for the test:
`var origParentDir = func() string {
	return filepath.Join("/etc", "machine-config-daemon", "orig")
}`
2. make package variables for the strings and have createOrigFile use those strings directly instead of the function calls ( I figured that would be "too much change" for no benefit )
3. have origParentDir/noOrigParentDir/createOrigFile take an path/prefix argument so I could redirect where they put their files ( again, I felt like that was too much change for no benefit) 
4. make package variables for the path strings and just have the existing functions return them 

I went with option 4, but I can do option 1 if that's preferable?